macos/FluentUI/Separator.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+@objc(MSFSeparatorStyle)
+public enum SeparatorStyle: Int {
+    case `default`
+    case shadow
+
+    fileprivate var color: NSColor {
+		switch self {
+		case .default:
+			if #available(OSX 10.14, *) {
+				return NSColor.separatorColor
+
+			} else {
+				return NSColor.systemGray
+			}
+		case .shadow:
+			return NSColor.gridColor
+        }
+    }
+}
+
+@objc(MSFSeparatorOrientation)
+public enum SeparatorOrientation: Int {
+    case horizontal
+    case vertical
+}
+
+
+@objc(MSFSeparator)
+open class Separator: NSView {
+	private var orientation: SeparatorOrientation = .horizontal
```

Comment:
similar here - private implies not mutable, so make it a let?

Reply:
Removed the var altogether